setup.py

```diff
@@ -7,10 +7,16 @@
 from __future__ import print_function
 
 import os
+imoprt sys
 from glob import glob
 
 from setuptools import setup, find_packages
 
+# ensure the current directory is on sys.path
+# so setupbase can be imported when pip uses
+# PEP 517/518 build rules.
+sys.path.append(os.path.dirname(__file__))
```

Comment:
Apply the same fix than for versioneer there: https://github.com/dask/dask-labextension/pull/170/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

We should maybe switch to `jupyter-packaging` rather than a local `setupbase` file?

Reply:
Yes, but we would need to check carefully, cause I'm wondering if there might have been some drift between the two.